### PR TITLE
rambler-co.ru 

### PR DESCRIPTION
**Pull request by SKEIDs**
https://github.com/AdguardTeam/AdguardFilters/issues/103578

### CODE REVIEW

File: SocialFilter/sections/specific.txt [hunk 1]
```diff
@@ -46,6 +46,7 @@ gpmradio.ru,originalnews.nico,archives.gov,sportnews.to,nationalchronicle.in,ana
 ! ##.sns_area
 sedaily.com,canshop.jp,koreatimes.co.kr,m.chosun.com,sports.donga.com##.sns_area
 !
+rambler-co.ru##.share__inner
```
Comment (Yuki2718) [inline SocialFilter/sections/specific.txt:49]: Why did you add this? Shouldn't this already covered by `##.share`?

Comment (SKEIDs) [reply SocialFilter/sections/specific.txt:49]: I made a mistake. sorry.

**Comment by SKEIDs** [comment]
@Yuki2718  I have a question unrelated to this pull request. can I ask?

**Review by zloyden — APPROVED** [review]

**Comment by Yuki2718** [comment]
> I have a question unrelated to this pull request. can I ask?

Sure, why not? You can ask here or alternatively can use my [personal issue thread](https://github.com/Yuki2718/adblock/issues/4).

**Comment by SKEIDs** [comment]
> > I have a question unrelated to this pull request. can I ask?
> 
> Sure, why not? You can ask here or alternatively can use my [personal issue thread](https://github.com/Yuki2718/adblock/issues/4).

thank you. I will be send question for your personal issue thread.
